src/js/popup/popup-new.js

```diff
@@ -1,3 +1,5 @@
+/* global */
+
```

Comment:
This doesn't mention any globals?

```suggestion
```

Reply:
Gahhh. Commented code noise. Good call out. Removed in a82836bffbe352126a8831b55a88a9b9c26a89da